### ISSUE TITLE
Prepare the sample for Quarkus on Azure Container Apps

### ISSUE THREAD

**Issue by saragluna** [Azure/azure-sdk-for-java#30938]
Since we already have the Java authentication plugins for MySQL and PostgreSQL, we should also prepare a sample about how to use it with Quarkus on Azure Container Apps.

**Comment by saragluna** [comment]
PR has been merged, still need to investigate how to run a native image with Quarkus.

**Comment by saragluna** [comment]
Need to update this doc https://learn.microsoft.com/azure/app-service/tutorial-java-quarkus-postgresql-app

**Comment by saragluna** [comment]
Native blocked by https://github.com/Azure/azure-sdk-for-java/issues/30332

**Comment by saragluna** [comment]
Closing this issue now, will create another issue for support native.
